src/Resolve/graphtype.jl

```diff
@@ -570,6 +570,19 @@ function wipe_snapshots!(graph::Graph)
     return graph
 end
 
+function color_string(c, str...)
+    if get(stderr, :color, false)  #check if color is allowed
```

Comment:
Use `sprint` with the `context` option.

https://github.com/JuliaLang/Pkg.jl/blob/f80d44345f61d0c1a02650f33322f9a330aa77d1/src/Operations.jl#L1661

Reply:
I can't use that because i don't have a `ctx` object, or an `IO` object
and I can't (in this case) use `stderr` in its place, not sure why but it just doesn't.

I could do:
```suggestion
    if Base.have_color  #check if color is allowed
```